src/index.html

```diff
@@ -5,9 +5,60 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article
+      class="product"
+      data-qa="card"
+    >
+      <div class="product__wrapper">
+
+        <div class="product__img">
+          <img
+            class="product__img-img"
+            src="./images/imac.jpeg"
+            alt="product-iamge"
```

Comment:
We know it is iMac image, so let's give this information to search bots.
```suggestion
            alt="Apple iMac"
```